exonum/src/node/mod.rs

```diff
@@ -1232,14 +1241,23 @@ mod tests {
         let db = Arc::from(Box::new(TemporaryDB::new()) as Box<dyn Database>) as Arc<dyn Database>;
         let node_cfg = helpers::generate_testnet_config(1, 16_500)[0].clone();
 
-        let services = vec![InstanceCollection::new(TestService).with_instance(
-            SERVICE_ID,
-            "test-service",
-            (),
-        )];
+        let service = TestService;
```

Comment:
This code snippet looks too low level, imo. You don't use power of type system to avoid some kind of boiler-plate code.

Reply:
I'd say it became more explicit according to previous review comments.